src/travis2docker/cli.py

```diff
@@ -94,7 +94,7 @@ def main(return_result=False):
     parser.add_argument(
         '--root-path',
         dest='root_path',
-        help="Root path to save scripts generated." "\nDefault: 'tmp' dir of your O.S.",
+        help="Root path to save scripts generated." f"\nDefault: {default_root_path}",
```

Comment:
I think this string is separated because of the recent implementation of autofixes e.g. black

But IMHO it should be separated, could we take advantage to fix it too, please?

```suggestion
        help=f"Root path to save scripts generated.\nDefault: {default_root_path}",
```